src/clients/base-evernode-client.js

```diff
@@ -410,7 +468,13 @@ class BaseEvernodeClient {
         return null;
     }
 
-    // To fetch records from host collection in Firestore database. (Pagination is applied - default page size :20)
+    /**
+     * Get all the hosts registered in Evernode in paginated manar. The result's are paginated. Default page size is 20. Note: Specifing both filter and pagination does not supported.
```

Comment:
registered hosts
This returns a  paginated result. (pagination has been mentioned twice)

Reply:
manner*